### PR TITLE
Create auto cs-fix GithubAction

### PR DESCRIPTION
**Pull request by staabm**
https://github.com/stefanzweifel/git-auto-commit-action/releases/v3.0.0

Todo: 
- [x] Personal Access Token, damit die GHAction triggered commits auch die Builds starten

### CODE REVIEW

File: .github/workflows/csfix.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.head_ref }}
+
+    - name: Run php-cs-fixer
+      uses: docker://oskarstark/php-cs-fixer-ga
+
+    - uses: stefanzweifel/git-auto-commit-action@v3.0.0
+      env:
+        GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```
Comment (localheinz) [inline .github/workflows/csfix.yml]: I have not yet tried providing a secret with the name `GITHUB_TOKEN`, so I can't tell whether that one will take precedence over the `GITHUB_TOKEN` that is automatically provided - perhaps it makes more sense to create a differently named token?

Comment (localheinz) [reply .github/workflows/csfix.yml]: Cool, looks like it worked: 25fefe481c142e16c74ce6fbc1331e3e0221c335!

Comment (staabm) [reply .github/workflows/csfix.yml]: I guess it didn't worked because the automated PR did not trigger new builds

Comment (localheinz) [reply .github/workflows/csfix.yml]: Ha, true!

Perhaps provide the secret as `STAAB_TOKEN`?

😉

Comment (staabm) [reply .github/workflows/csfix.yml]: yep, will provide a token ;). thx

Comment (localheinz) [reply .github/workflows/csfix.yml]: Let me know if I should adjust #23 accordingly!

**Pull request merged**